ml_tools/kerasmodel.py

```diff
@@ -1,10 +1,15 @@
+import math
 import os
 import json
 import logging
 import tensorflow as tf
-import cv2
 import numpy as np
-from ml_tools.dataset import TrackChannels
+from track.track import TrackChannels
+from classify.trackprediction import TrackPrediction
+from ml_tools.preprocess import (
+    preprocess_movement,
+    preprocess_frame,
+)
 
 
 class KerasModel:
```

Comment:
its just that it's tensorflow v2 keras framework rather than based of 1.15 as the old one, I imagine soon we can get rid of the old model code if we are happy with this